test/ScaledDomains.Extensions.Caching.MySql.Tests/MySqlServerCacheMaintenanceTests.cs

```diff
@@ -81,41 +90,41 @@ public async Task ExecuteAsync_ShouldDeleteExpiredCacheItems()
         {
             // Arrange
 
+            var utcNow = DateTimeOffset.UtcNow;
```

Comment:
it's just a comment nothing more :) if I implement an integration test I always use hardcoded datetime to avoid the precision differences.

Reply:
👍